shared/src/main/scala/mouse/MouseFunctions.scala

```diff
@@ -32,6 +30,10 @@ trait MouseFunctions {
    * @param a
    *   - the value to be evaluated and ignored.
    */
-  def ignore(@nowarn a: Any): Unit = ()
```

Comment:
According to the new behavior in Scala 2.13.9, this `@nowarn` was marked as unused.

> home/runner/work/mouse/mouse/shared/src/main/scala/mouse/MouseFunctions.scala:35:15: @nowarn annotation does not suppress any warnings
[error]   def ignore(@nowarn a: Any): Unit = ()

And it's done [intentionally](https://github.com/scala/bug/issues/12648#issuecomment-1257010205). So we had to handle this issue with some workaround.

Reply:
@benhutchison your thoughts on this would be appreciated.

Reply:
Annoying and ironic that suppressing warnings generates a warning!

But if there's no warning, can we remove the annotation?

Reply:
This new behavior is only for 2.13.9. So, if we just remove the annotation, then for 2.12 and 3 that will cause warnings about unused function argument.